source/istio_gateway.go

```diff
@@ -35,7 +35,10 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/cache"
 
+	antns "sigs.k8s.io/external-dns/source/annotations"
```

Comment:
Why rename? It reads simpler if you don't alias the package import here.

Reply:
Removed `antns`. The main reason was - `annotations` is quite a common package name in most of the projects.

Reply:
Example method attributes https://github.com/kubernetes-sigs/external-dns/blob/277e8ca2ec7e10f749083bfd9842d61cc4fd8f32/source/source.go#L247. There was shadowing. So renamed to `input`